pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -148,8 +148,10 @@ func (os *OpenStack) CreateVolume(name string, size int, tags *map[string]string
 	}
 
 	opts := volumes.CreateOpts{
-		Name: name,
-		Size: size,
+		Name:         name,
+		Size:         size,
+		VolumeType:   vtype,
```

Comment:
I agree with @rootfs, volume types and zones are defined by OpenStack admin after installation and `""` is the only one we can be sure it is there.